src/mono/mono/mini/mini-llvm.c

```diff
@@ -563,6 +568,14 @@ create_address (MonoLLVMModule *module, LLVMValueRef value, LLVMTypeRef type)
 	return res;
 }
 
+static Address*
+create_address_from_value (MonoLLVMModule *module, LLVMValueRef value)
+{
+	LLVMTypeRef etype = LLVMGetElementType (LLVMTypeOf (value));
```

Comment:
Just to make sure I understand. This function will go away once we're fully on LLVM 14, but for now you're using it in places where we still have `LLVMValueRef` instead of an `Address`?

Reply:
Places using this still need to be fixed to work with opaque pointers.

Reply:
Fixed it.